dotCMS/src/main/java/com/dotcms/rest/api/v1/browser/BrowserQueryForm.java

```diff
@@ -0,0 +1,262 @@
+        this.showWorking   = builder.showWorking;
+        this.showArchived  = builder.showArchived;
+        this.showFolders   = builder.showFolders;
+        this.showFiles     = builder.showFiles;
+        this.sortByDesc    = builder.sortByDesc;
+        this.showLinks     = builder.showLinks;
+        this.showPages     = builder.showPages;
+        this.showDotAssets = builder.showDotAssets;
+        this.languageId    = builder.languageId;
+    }
+
+    public String getHostFolderId() {
+        return hostFolderId;
+    }
+
+    public String getFilter() {
+        return filter;
+    }
+
+    public String getSortBy() {
+        return sortBy;
+    }
+
+    public int getOffset() {
+        return offset;
+    }
+
+    public int getMaxResults() {
+        return maxResults;
+    }
+
+    public boolean isShowWorking() {
+        return showWorking;
+    }
+
+    public boolean isShowArchived() {
+        return showArchived;
+    }
+
+    public boolean isShowFolders() {
+        return showFolders;
+    }
+
+    public boolean isShowFiles() {
+        return showFiles;
+    }
+
+    public boolean isShowPages() {
+        return showPages;
+    }
+
+    public boolean isSortByDesc() {
+        return sortByDesc;
+    }
+
+    public boolean isShowLinks() {
+        return showLinks;
+    }
+
+    public boolean isShowDotAssets() {
+        return showDotAssets;
+    }
+
+    public long getLanguageId() {
+        return languageId;
+    }
+
+    public List<String> getExtensions() {
+        return extensions;
+    }
+
+    public List<String> getMimeTypes() {
+        return mimeTypes;
+    }
+
+    @Override
+    public String toString() {
+        return "BrowserQueryForm{" +
+                "hostFolderId='" + hostFolderId + '\'' +
+                ", filter='" + filter + '\'' +
+                ", sortBy='" + sortBy + '\'' +
+                ", offset=" + offset +
+                ", maxResults=" + maxResults +
+                ", showWorking=" + showWorking +
+                ", showArchived=" + showArchived +
+                ", showFolders=" + showFolders +
+                ", showFiles=" + showFiles +
+                ", showPages=" + showPages +
+                ", sortByDesc=" + sortByDesc +
+                ", showLinks=" + showLinks +
+                ", showDotAssets=" + showDotAssets +
+                ", languageId=" + languageId +
+                ", extensions=" + extensions +
+                ", mimeTypes=" + mimeTypes +
+                '}';
+    }
+
+    public static final class Builder {
+
+        @JsonProperty
+        private String hostFolderId   = Folder.SYSTEM_FOLDER;
+
+        @JsonProperty
+        private String filter         = null;
+
+        @JsonProperty
+        private List<String> mimeTypes  = new ArrayList<>();
+
+        @JsonProperty
+        private List<String> extensions = new ArrayList<>();
+
+        @JsonProperty
+        private String sortBy         = "moddate";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field sortBy has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)